library/Zend/Db/Statement.php

```diff
@@ -198,6 +198,12 @@ protected function _stripQuoted($sql)
             $sql = preg_replace("/$q([^$q{$escapeChar}]*|($qe)*)*$q/s", '', $sql);
         }
 
+        if ($sql === null) {
+            // this preg_replace may return NULL in case of error (PREG_BACKTRACK_LIMIT_ERROR).
+            // In that case the result of this method will be an empty string.
```

Comment:
```suggestion
            // this preg_replace call can return NULL in case of error (PREG_BACKTRACK_LIMIT_ERROR).
            // In this case, the result of this method will be an empty string.
```

Reply:
fixed in internal repo